vars/parallelDockerUpdatecli.groovy

```diff
@@ -30,9 +41,14 @@ def call(userConfig = [:]) {
     },
     'updatecli': {
       withCredentials([string(credentialsId: finalConfig.credentialsId,variable: 'UPDATECLI_GITHUB_TOKEN')]) {
-        updatecli(action: 'diff', containerMemory: finalConfig.containerMemory)
+        // Merging the 2 maps - https://blog.mrhaki.com/2010/04/groovy-goodness-adding-maps-to-map_21.html
+        Map updatecliConfig = finalConfig.updatecliConfig << [action: 'diff']
+
+        updatecli(updatecliConfig)
         if (env.BRANCH_IS_PRIMARY) {
-          updatecli(action: 'apply', cronTriggerExpression: finalConfig.cronTriggerExpression, containerMemory: finalConfig.containerMemory)
+          // Merging the 2 maps - https://blog.mrhaki.com/2010/04/groovy-goodness-adding-maps-to-map_21.html
+          updatecliConfig = finalConfig.updatecliConfig << [action: 'apply', cronTriggerExpression: '@weekly']
```

Comment:
```suggestion
          updatecliConfig = finalConfig.updatecliConfig << [action: 'apply', cronTriggerExpression: finalConfig.updatecliConfig.cronTriggerExpression]
```

Reply:
if `updatecliConfig` is null, then the suggestion triggers a java error stack (NullPointer)